recipes/aws-noaa-outgoing-longwave-radiation-monthly/meta.yaml

```diff
@@ -0,0 +1,22 @@
+title: 'AWS NOAA OUTGOING LONGWAVE RADIATION MONTHLY'
+description: 'Analysis-ready datasets derived from NOAA OUTGOING LONGWAVE RADIATION MONTHLY'
```

Comment:
can you fill in descriptions from the documentation webpages instead of using the title in this field?

https://www.ncei.noaa.gov/products/climate-data-records/outgoing-longwave-radiation-daily